tests/rkt_run_pod_manifest_test.go

```diff
@@ -586,6 +586,132 @@ func TestPodManifest(t *testing.T) {
 			fmt.Sprintf("%v=enabled", capability.CAP_NET_ADMIN.String()),
 			"",
 		},
+		{
+			// Set valid non-numerical app user and group.
+			[]imagePatch{
+				{"rkt-test-run-pod-manifest-valid-user-group.aci", []string{}},
+			},
+			&schema.PodManifest{
+				Apps: []schema.RuntimeApp{
+					{
+						Name: baseAppName,
+						App: &types.App{
+							Exec:  []string{"/inspect", "--print-user"},
+							User:  "user1",
+							Group: "group1",
```

Comment:
Does it actually work when uid != gid? (in this case 1000!=100)

I remember having problems with this. But it's great if it works :)